migration/migrator/data/submitty_db.sql

```diff
@@ -51,6 +51,7 @@ CREATE TABLE courses (
 CREATE TABLE emails (
     id serial NOT NULL,
     user_id character varying NOT NULL,
+    recipient  varchar(255) NOT NULL,
```

Comment:
This probably shouldn't be re-added? (Removed in #4148)

Reply:
I haven't actually touched the emails table.  Not sure how/why that is re-added.  But yeah — I'll take it back out.